ChibiOS/BrainPad2/board.h

```diff
@@ -71,7 +73,7 @@
 /*
  * MCU type as defined in the ST header.
  */
-#define STM32F411xE
+#define STM32F401xE
```

Comment:
Are you sure about this? Is the MCU different from the one described on the README?

Reply:
It would not build with the F411 complaining that it was the incorrect mcuconf. The mcu above allows the build and matches the series on mouser and micropython (as far as I can tell).

Reply:
See [previous ](https://github.com/nanoframework/nf-Community-Targets/pull/193/files#r769550796)